frontend/src/components/bauraten/BauratenAggregiertComponent.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <div v-if="showResult">
+    <field-group-card :card-title="header">
+      <v-row justify="center">
+        <v-col cols="12">
+          <v-container class="table">
+            <v-data-table
+              :headers="bauratenJahreHeaders"
+              :items="aggregatedBauraten"
+              hide-default-footer
+            >
+              <template #header="{ text }">
+                <span>{{ text }}</span>
+              </template>
+              <template #body="{ items }">
+                <tbody>
+                  <tr>
+                    <td><span>Wohneinheiten</span></td>
+                    <td
+                      v-for="(item, index) in items"
+                      :key="index"
+                    >
+                      {{ item.anzahlWeGeplant }}
+                    </td>
+                  </tr>
+                  <tr>
+                    <td><span>Geschossfläche</span></td>
+                    <td
+                      v-for="(item, index) in items"
+                      :key="index"
+                    >
+                      {{ item.geschossflaecheWohnenGeplant }}
+                    </td>
+                  </tr>
+                </tbody>
+              </template>
+            </v-data-table>
+          </v-container>
+        </v-col>
+      </v-row>
+    </field-group-card>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from "vue-property-decorator";
+import FieldGroupCard from "@/components/common/FieldGroupCard.vue";
+import AbfragevarianteModel from "@/types/model/abfragevariante/AbfragevarianteModel";
+import BauabschnittModel from "@/types/model/bauabschnitte/BauabschnittModel";
+import BaugebietModel from "@/types/model/baugebiete/BaugebietModel";
+import BaurateModel from "@/types/model/bauraten/BaurateModel";
+import { DataTableHeader } from "@/types/common/DataTableHeader";
+import { BauabschnittDto, BaugebietDto, BaurateDto } from "@/api/api-client/isi-backend";
+import _ from "lodash";
+
+@Component({ components: { FieldGroupCard } })
+export default class BauratenAggregiertComponent extends Vue {
+  @Prop()
+  private aggregateBauraten!: AbfragevarianteModel | BauabschnittModel | BaugebietModel;
+
+  get showResult(): boolean {
+    return this.aggregatedBauraten.length > 0;
+  }
+
+  private baurateMap: Map<number, BaurateModel> = new Map<number, BaurateModel>();
+
+  /**
+   * Erstellt eine Map mit Bauraten zu dem angegebenen Property aggregateBauraten
```

Comment:
Gehört der Kommentar zur Methode `aggregatedBauraten`? Falls ja die Leerzeile in Zeile 71 entfernen.